### PR TITLE
Pin fastparquet

### PR DESCRIPTION
**Pull request by lithomas1**
- [ ] xref #42588
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#code-standards) for how to run them
- [ ] whatsnew entry

Get CI to green.

### CODE REVIEW

**Comment by bashtage** [comment]
Should probably pin in requirements-dev.txt as well until it is ready to move to 0.7.0.

**Comment by simonjayhawkins** [comment]
The fastparquet tests also fail on 1.3.x, so milestone 1.3.1 to trigger backport

**Comment by simonjayhawkins** [comment]
@lithomas1 can you open an issue to unpin this.

**Review by simonjayhawkins — CHANGES_REQUESTED** [review]
pre-commit failure.

**Comment by simonjayhawkins** [comment]
> @lithomas1 can you open an issue to unpin this.

added xref to the issue to the op, so no need to open a separate issue.

File: ci/deps/actions-38-db.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ dependencies:
   - beautifulsoup4
   - botocore>=1.11
   - dask
-  - fastparquet>=0.4.0
+  - fastparquet>=0.4.0, < 0.7.0
```
Comment (jreback) [inline ci/deps/actions-38-db.yaml:18]: this causes a pre-commit failure (remove the space before the 0.7.0)

**Comment by lithomas1** [comment]
@github-actions pre-commit

**Comment by lithomas1** [comment]
Green, can someone from @pandas-dev/pandas-core merge?

Sorry about the noise in this PR, was using the Github Web UI, which pre-commit apparently doesn't like.

**Review by phofl — APPROVED** [review]

**Comment by phofl** [comment]
thx @lithomas1

**Comment by phofl** [comment]
backported manually

**Comment by lithomas1** [comment]
thx @phofl.

**Pull request merged**